ddlitlab2024/dataset/cli/run.py

```diff
@@ -25,27 +27,39 @@ def main():
     try:
         logger.debug("Parsing CLI args...")
         args: Namespace = CLIArgs().parse_args()
-        logger.debug(f"CLI args: {args}")
+        logger.debug(f"Parsed CLI args: {args}")
 
         if args.version:
             logger.info(f"running ddlitlab2024 CLI v{__version__}")
             sys.exit(0)
 
         if args.command == CLICommand.DB:
             create_schema = args.db_command == "create-schema" or args.db_command == "dummy-data"
-            db: Session = Database(args.db_path).create_session(create_schema=create_schema)
-            logger.debug("Database session created")
+            db: Database = Database(args.db_path).create_session(create_schema=create_schema)
 
             match args.db_command:
                 case "recording2mcap":
                     from ddlitlab2024.dataset.recording2mcap import recording2mcap
 
-                    recording2mcap(db, args.recording, args.output_dir)
+                    recording2mcap(db.session, args.recording, args.output_dir)
 
                 case "dummy-data":
                     from ddlitlab2024.dataset.dummy_data import insert_dummy_data
 
-                    insert_dummy_data(db, args.num_recordings, args.num_samples_per_rec, args.image_step)
+                    insert_dummy_data(db.session, args.num_recordings, args.num_samples_per_rec, args.image_step)
+
+        elif args.type == ImportType.ROS_BAG:
+            db: Database = Database(args.db_path).create_session()
+            logger.info(f"Trying to import file '{args.file}' to database...")
+            metadata = ImportMetadata(
+                allow_public=True,
+                team_name="Bit-Bots",
```

Comment:
Okay, it's fine for now. We can create args for this later on

Reply:
Yeah, I think this makes sense to do once we have a seperate data source.
Then we can decide if we want to use CLI args or a config yaml file or something of the sort.